### DISCUSSION TITLE
请问大佬们，iOS仓使用薄荷中小鹤双拼怎么触发音形

### DISCUSSION THREAD

**Discussion by BigfaceCats** [Mintimate/oh-my-rime#85] [Q&A] [answered]
我在仓设置中打开了键盘的分号，但是这个分号会导致候选字直接上墙

**Comment by Mintimate** [comment, marked as answer]
默认布局的情况，把目标触发键改为“经过RIME处理”之后就可以用来。如果是自定义布局也是一样的。

![效果](https://github.com/Mintimate/oh-my-rime/assets/43734212/5e316537-9964-4a7f-8019-86ed46e7a291)

**Reply by BigfaceCats** [reply]
大佬，我这边用的是“小鹤双拼-薄荷定制”方案，用你这个方法还是不行，但是可以在默认布局开启分号按键实现。请问是被什么设置强行覆盖了吗？
PS：另外羡慕大佬手机显示形码，我开启了    show_aux_notice: "always"手机也不显示形码？这是怎么做到的
<img src="https://github.com/Mintimate/oh-my-rime/assets/24986603/d11c0a8f-3d41-4e33-a8c0-16132c19652e" width="30%">

**Reply by Mintimate** [reply]
你用文档里的皮肤试试看，分号问题不清楚了，更新为薄荷最新代码了么？
